privaterelay/templates/includes/premium-onboarding.html

```diff
@@ -70,18 +71,18 @@ <h2>{% ftlmsg 'multi-part-onboarding-premium-welcome-subheadline' %}</h2>
         <li class="c-premium-onboarding-step c-premium-onboarding-step-3 {% if user_profile.onboarding_state == 2 %} is-visible {% endif %}">
             <div class="mzp-l-content mzp-t-content-lg mzp-t-content-nospace">
                 <div class="c-premium-onboarding-headlines">
-                    <h1>{% ftlmsg 'multi-part-onboarding-premium-extension-headline' %}</h1>
-                    <h2>{% ftlmsg 'multi-part-onboarding-premium-welcome-subheadline' %}</h2>
+                    <h1>{% ftlmsg 'multi-part-onboarding-reply-headline' %}</h1>
                 </div>
 
                 <div class="c-premium-onboarding-content">
                     <div class="c-premium-onboarding-content-image">
-                        <img src="{% static 'images/dashboard-onboarding/woman-email.svg' %}" alt="Woman next to large envelope">
+                        <img src="{% static 'images/dashboard-onboarding/man-laptop-email-alt.svg' %}" alt="Woman next to large envelope">
                     </div>
                     <div class="c-premium-onboarding-content-description">
-                        <p><strong>{% ftlmsg 'multi-part-onboarding-premium-extension-reply-title' %}</strong> {% ftlmsg 'onboarding-premium-reply-description'%}</p>
+                        <p class="c-premium-onboarding-detail">{% ftlmsg 'onboarding-premium-title-detail' %}</p>
                         <div class="c-premium-onboarding-install-addon">
-                            <p><strong>{% ftlmsg 'multi-part-onboarding-premium-extension-get-title' %}</strong> {% ftlmsg 'multi-part-onboarding-premium-extension-get-description' %}</p>
+                            <p><strong>{% ftlmsg 'onboarding-premium-reply-title' %}</strong> {% ftlmsg 'onboarding-premium-reply-description' %}                
```

Comment:
Blocking: I think what we want to do here is show the new strings on small screens, but the old strings on larger screens. This is because on larger screens, we show a "Get Relay Extension" button, so text about getting the extension makes sense, whereas on smaller screens, we don't, because the extension is not available there.

Reply:
Good call! I added a mobile/desktop condition to show different types of copy.

https://user-images.githubusercontent.com/13066134/147238298-95280e9d-1370-4838-85d8-d25a9b904290.mov

Reply:
https://user-images.githubusercontent.com/13066134/147238552-04ba4a44-9fd6-4ebb-8b15-3d91b861c959.mov